html/mycakeapp/src/Model/Table/BiditemsTable.php

```diff
@@ -95,7 +110,17 @@ public function validationDefault(Validator $validator)
      */
     public function buildRules(RulesChecker $rules)
     {
-        $rules->add($rules->existsIn(['user_id'], 'Users'));
+        $rules
+            ->add($rules->existsIn(['user_id'], 'Users'));
+
+        $rules
+            ->add(function ($entity) {
+                $result = (bool)preg_match('/\.gif$|\.png$|\.jpg$|\.jpeg$/i', $entity['image_name']);
+                return $result;
+            }, 'isCheck', [
```

Comment:
```suggestion
            }, 'extension', [
```
ルール名がisCheckは何のチェックか分からないのでextensionにしましょう